tickit/devices/sink.py

```diff
@@ -9,28 +9,28 @@
 
 
 class Sink(ConfigurableDevice):
-    """A simple device which can take any input and produces no output"""
+    """A simple device which can take any input and produces no output."""
 
     #: A typed mapping containing the 'input' input value
     Inputs: TypedDict = TypedDict("Inputs", {"input": Any})
     #: An empty typed mapping of device outputs
     Outputs: TypedDict = TypedDict("Outputs", {})
 
     def __init__(self) -> None:
-        """A constructor of the sink, with no arguments"""
+        """A constructor of the sink, with no arguments."""
         pass
 
     def update(self, time: SimTime, inputs: Inputs) -> DeviceUpdate[Outputs]:
-        """The update method which prints the inputs to console and produces no outputs
+        """The update method which logs the inputs at debug level and produces no outputs.
```

Comment:
`time` and `inputs` should be included in the docstring

Reply:
These are already there?